addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -206,6 +206,10 @@ function factory(dependencies) {
          * @returns {mail.chat_window|undefined}
          */
         _getNextVisibleUnfoldedChatWindow({ reverse = false } = {}) {
+            if (this.env.messaging.locale.textDirection === 'rtl') {
+                reverse = !reverse;
+            }
```

Comment:
This is not good. The notions of "next" and "reverse" stay the same, regardless of text direction.
The processing may change, but it doesn't make sense to modify `reverse`, in fact it's bad practice to modify in place parameters.